org.gnome.siglo.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id" : "org.gnome.siglo",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "siglo",
+    "finish-args" : [
+        "--allow=bluetooth",
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=system-bus",
```

Comment:
Please use `--system-talk-name` to specify the dbus name.

Reply:
not sure what you mean by this. it uses org.bluez service on the System Bus. so i need to do something like:
`--system-talk-name=org.bluez` instead of `--socket=system-bus` ?

Reply:
Exactly.